CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+This page explains common tasks related to working with Xtext's source code.
+
+## Report an Issue
+
+Please create a GitHub issue in this repository.
```

Comment:
Nitpicky: `this repository` might be misleading if you have a local clone / fork. I'd put a link to the new issue page behind `issue`.

Reply:
Done